jobs/yoyowallet_backend_platform_engineer.html

```diff
@@ -0,0 +1,60 @@
+---
+title: Backend Platform Engineer
+company: Yoyo Wallet
+url: https://boards.greenhouse.io/yoyowallet/jobs/570082
+location: London, England
+contract: permanent
+contact:
+    name: Jian Yuan Lee
+    email: jian@yoyowallet.com
+created: !!timestamp '2017-02-17'
+tags:
+  - python
+  - sql
+  - django
+  - postgresql
+  - rabbitmq
+  - celery
+  - docker
+  - aws
+---
+
+# Yoyo Wallet
+
+Launched in 2014 at Imperial College London, Yoyo Wallet has become the UK’s fastest growing mobile wallet by expanding across hundreds of University and Corporate locations. In the process Yoyo has developed a unique platform that opens up a global opportunity to power seamless payment and loyalty experiences for retailers, developers and financial institutions across the world. Founded by Alain Falys and Michael Rolph, Yoyo has raised significant funding ($15m as of September 2015) and built a team of energetic entrepreneurs who know how to get things done. With so much potential ahead, Yoyo is looking for a Backend Engineer to join our team.
+
+If you are looking to join a fun, fast growing FinTech firm, keep reading. We were recently chosen by KPMG as one of 50 FinTechs to watch globally. Now in our 4th year, we are in over 40 UK universities, over 100 corporate locations and have a global presence with business in the US, Spain, Ireland and Singapore.
+
+The Yoyo Wallet product comprises of iOS and Android apps that talk to a suite of APIs powered by the Yoyo platform. We use a service-oriented architecture to support real-time, high-volume transactions that consistently deliver sub-one-second response times at the point-of-sale.
+
+**Backend stack:** includes (but is not limited to): Python, Django, Event messaging and RESTful APIs, Micro-Services Oriented Architecture, PostgreSQL, DynamoDB, RabbitMQ, Celery, Puppet, Fabric, Docker, CircleCI / Continuous Deployment via ChatOps, and is hosted on AWS.
```

Comment:
If these are important skills for your new hire you can add them to the list of tags.